doc/snippets/Microsoft.Data.SqlClient/SqlConfigurableRetryFactory.xml

```diff
@@ -0,0 +1,98 @@
+|10053|Could not convert the data value due to reasons other than sign mismatch or overflow.|
+|997|A connection was successfully established with the server, but then an error occurred during the login process. (provider: Named Pipes Provider, error: 0 - Overlapped I/O operation is in progress)|
+|233|A connection was successfully established with the server, but then an error occurred during the login process. (provider: Shared Memory Provider, error: 0 - No process is on the other end of the pipe.) (Microsoft SQL Server, Error: 233)|
+|64||
+|20||
+|0||
+
+]]>
+            </format>
+            </remarks>
+        </SqlConfigurableRetryFactory>
+        <CreateExponentialRetryProvider>
+            <param name="retryLogicOption">An object of <see cref="T:Microsoft.Data.SqlClient.FloatingRetryLogicOption" /> carries the configurations for the object creation.</param>
+            <summary>Provides an exponential time interval retry logic provider.</summary>
+            <returns>A <see cref="T:Microsoft.Data.SqlClient.SqlRetryLogicBaseProvider" /> object.</returns>
+            <remarks>
+				<format type="text/markdown"><![CDATA[
+
+> [NOTE!]
+> The inner enumerator includes the randomization to prevent multiple instances of the client sending subsequent retry attempts at the same time.
+
+]]></format>
+			</remarks>
+            <exception cref="T:System.ArgumentNullException">If the `retryLogicOption` parameter was null.</exception>
+        </CreateExponentialRetryProvider>
+        <CreateIncrementalRetryProvider>
+            <param name="retryLogicOption">An object of <see cref="T:Microsoft.Data.SqlClient.FloatingRetryLogicOption" /> carries the configurations for the object creation.</param>
+            <summary>Provides an incremental time interval retry logic provider.</summary>
+            <returns>A <see cref="T:Microsoft.Data.SqlClient.SqlRetryLogicBaseProvider" /> object.</returns>
+            <remarks>
+				<format type="text/markdown"><![CDATA[
+
+> [NOTE!]
+> The inner enumerator includes the randomization to prevent multiple instances of the client sending subsequent retry attempts at the same time.
+
+]]></format>
+			</remarks>
+            <exception cref="T:System.ArgumentNullException">If the `retryLogicOption` parameter was null.</exception>
+        </CreateIncrementalRetryProvider>
+        <CreateFixedRetryProvider>
+            <param name="retryLogicOption">An object of <see cref="T:Microsoft.Data.SqlClient.FixedRetryLogicOption" /> carries the configurations for the object creation.</param>
+            <summary>Provides a fixed interval time retry logic provider.</summary>
+            <returns>A <see cref="T:Microsoft.Data.SqlClient.SqlRetryLogicBaseProvider" /> object.</returns>
+            <remarks>
+				<format type="text/markdown"><![CDATA[
+
+> [NOTE!]
+> The inner enumerator includes the randomization to prevent multiple instances of the client sending subsequent retry attempts at the same time.
```

Comment:
```suggestion
> The inner enumerator includes randomization to prevent multiple instances of the client from performing subsequent retry attempts at the same time.
```